packages/twenty-front/src/modules/object-record/components/RecordChip.tsx

```diff
@@ -47,6 +59,25 @@ export const RecordChip = ({
 
   const recordIndexOpenRecordIn = useRecoilValue(recordIndexOpenRecordInState);
 
+  const isSidePanelViewOpenRecordInType =
+    recordIndexOpenRecordIn === ViewOpenRecordInType.SIDE_PANEL;
+
+  const handleCustomClick = isSidePanelViewOpenRecordInType
+    ? () =>
+        openRecordInCommandMenu({
+          recordId: record.id,
+          objectNameSingular,
+        })
+    : undefined;
+
+  const { onClick, onMouseDown } = useMouseDownNavigation({
+    to: to ?? getLinkToShowPage(objectNameSingular, record),
+    onClick: handleCustomClick,
+    disabled: forceDisableClick,
+    stopPropagation: true,
```

Comment:
open-question: I'm not a fan of stopping event propagation. Sometime we haven't found a work around yet but in the idea as the app has a state we can handle it at react level (with early return in the other onClick handler for instance). While we don't want to change it here, I'm not sure it's a good idea to add it as part of the useMouseDownNavigation hook as if it was a good idea to use it. I would remove it from there, and locally prevent propagation in the usage of onClick below

Reply:
Removed stopPropagation to do it locally